tpdatasrc/tpgamefiles/mes/stat_ext.mes

```diff
@@ -295,6 +295,19 @@ Feats: Spell Focus (conjuration).
 Spells: Able to cast lesser planar ally.}
 
 {13027}Favored Soul{Favored souls cast divine spells by means of an innate connection rather than through laborious training and prayer, so their divine connection is natural rather than learned.}
+
+{13031}Stormlord{Stormlords often live as brigands, indulging their personal desires for wealth, food, luxury items, and wanton behavior as they crave random, spectacular acts of violence.
```

Comment:
The desc should say something about what the class is mechanically, this is just fluff description.